test/ts/unit/simple_interest_terms_contract.ts

```diff
@@ -612,9 +612,11 @@ contract("SimpleInterestTermsContract (Unit Tests)", async (ACCOUNTS) => {
 
             const ORIGIN_MOMENT = moment();
             const BLOCK_ISSUANCE_TIMESTAMP = ORIGIN_MOMENT.unix();
+            const INTEREST_RATE_SCALING_FACTOR = new BigNumber(10 ** 4);
```

Comment:
use the adapter's scaling factor because this is a constant shared across the system.